.editorconfig

```diff
@@ -4,6 +4,8 @@
 
 [*.cs]
 
+file_header_template = Copyright (c) Microsoft Corporation and Contributors\r\nLicensed under the MIT license.

```

Comment:
Note that the copyright text used is picked from StyleCop.

https://github.com/microsoft/devhome/blob/524850efa92eba1f0e9249c77d17979bc8c91186/codeAnalysis/StyleCop.json#L6

Some file are misaligned due to StyleCop rule SA1636 being disabled (fixed in this PR).
https://github.com/microsoft/devhome/blob/524850efa92eba1f0e9249c77d17979bc8c91186/codeAnalysis/GlobalSuppressions.cs#L60

If copyright text needs to be aligned, if possible, I would prefer to do this in a separated PR since this is already big 😃

Reply:
I'm sorry this took a while. Looks like we do need to change it, although I agree that should be a separate commit. Can you remove this line from here though? I don't think we need to undo all the changes it made in the rest of the code, but I don't want to introduce this somewhere new.

Reply:
Reverted and aligned with main branch.